### PR TITLE
feat: Set default ir to dynamo export

### PR DESCRIPTION
**Pull request by peri044**
BREAKING CHANGE: Changing default IR from Torchscript to dynamo export

# Description

Changing default IR from Torchscript to dynamo export

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

**Comment by narendasan** [comment]
@peri044 Bump the version to 2.0 in this PR so its clear this is a breaking change

File: py/torch_tensorrt/_compile.py [hunk 1]
```diff
@@ -61,17 +61,17 @@ def _get_target_ir(module_type: _ModuleType, ir: str) -> _IRType:
...
+                    logging.Level.Info, "ir was set to default, using dynamo as ir"
                 )
-                return _IRType.ts
-            elif module_is_fxable:
+                return _IRType.dynamo
+            elif module_is_tsable:
                 logging.log(
                     logging.Level.Warning,
-                    "Input graph is a torch.fx.GraphModule but the ir provided is default (ts). Please set ir=dynamo to suppress the warning.",
+                    "Input graph is a Torchscript module but the ir provided is default (dynamo). Please set ir=torchscript to suppress the warning. Compiling the module with ir=ts",
```
Comment (gs-olive) [inline py/torch_tensorrt/_compile.py:72]: Very minor change - make `ir=ts` or`ir=torchscript` at both locations in the log message.

**Review by gs-olive — APPROVED** [review]
Looks good to me. Made a small comment on log messaging.

File: py/torch_tensorrt/_compile.py (hunk 1 above)
Comment (peri044) [reply py/torch_tensorrt/_compile.py]: Fixed it

**Review by narendasan — APPROVED** [review]
LGTM pending CI

**Pull request merged**